mitosheet/css/taskpanes/Graph/GraphSidebar.css

```diff
@@ -93,6 +110,45 @@
     text-align: center;
 }
 
+.graph-element-popup-div.popup-div:before {
+    content: "";
+    display: block;
+    height: 10px;
+    width: 10px;
+    right: 50%;
+    margin-left: 0;
+    margin-top: -5px;
+    background: var(--mito-highlight-medium);
+    position: absolute;
+    top: 0;
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    transform: rotate(45deg);
+    z-index: -1;
+}
+
+.graph-element-popup-div-caret-below-left.graph-element-popup-div.popup-div:before {
+    top: 100%;
+    left: 12px;
+}
+
+.graph-element-popup-div-caret-below-centered.graph-element-popup-div.popup-div:before {
+    top: 100%;
+    left: 70px;
+}
+
+.graph-element-popup-div-caret-above.graph-element-popup-div.popup-div:before {
+    top: 0;
+}
+
+.graph-element-popup-div.popup-div {
+    padding: 3px;
+    background: var(--mito-highlight-medium);
+    border-radius: 3px;
+}
```

Comment:
I think we can make this look a little bit more like the mockups if we change to the darker purple `var(--mito-highlight)`

<img width="350" alt="Screenshot 2024-01-25 at 2 35 22 PM" src="https://github.com/mito-ds/mito/assets/18709905/44f6c352-f0dc-485f-837b-c06925ffe818">